pom.xml

```diff
@@ -21,6 +21,7 @@
     <checkstyle.version>10.6.0</checkstyle.version>
     <lifecycle-mapping.version>1.0.0</lifecycle-mapping.version>
     <jjwt.version>0.11.5</jjwt.version>
+    <jackson.version>${jackson-bom.version}</jackson.version>
```

Comment:
what reads this variable?

Reply:
theoretically that jjwt pom.xml. That's where this property is defined (to an older version of jackson) Theoretically oif we define the same property in the main pom, it will override the properties in the dependent poms.

Intrestingly, when I built a standalona jar from it (without this extra property) the created jar only contained the latest version of the jackson jar.

So maybe maven automatically recognises the different varsions and keeps only the newest - and it is listed on the maven central twice becasue it doesn't do the same.
... or the stanalone jar builder do that.